### DISCUSSION TITLE
[OUTDATED] Auto-Sanctions

### DISCUSSION THREAD

**Discussion by andreaskapsalhs** [Shadow-Creation-Team/Protector#32] [Announcement]
# Hello everyone!

## It came to my attention that the following issues are addressing pretty much the same concept:

 - > [Anti Spam](https://github.com/ShadyDevelopers/Shady/issues/3)
 - > [Auto Sanctions](https://github.com/ShadyDevelopers/Shady/issues/22)
 - > [Dynamic Slowmode](https://github.com/ShadyDevelopers/Shady/issues/21)

So to tidy-up those issues, this discussion is about describing the exact feature that's included in **v0.5**.
___

By the release of the upcoming **v0.5** including the Dashboard, we are going to add a new way to moderate your server with minimal effort. The new feature will be called **Auto Sanctions** and it will be under the **Moderation** tab in the dashboard. Customized to suit your server needs, you'll be able to set values for actions such as:

- Blacklisted words / links in specific channels
- Anti-spam with specific message interval

All with the option to warn the user **automatically**.

___

## Also, you'll be able to set a punishment system via the dashboard. An example:

- At 3 bot's warns » Mute for a specific reason.
- At 5 bot's warns » Remove x roles from user.
- At 2 bot's mutes » Ban from the server for x days.
- etc...
___

## Current Progression:
- [x] AntiSpam
- [x] AutoInfractions
- [ ] AntiAlt
- *more to come..*

___

We are really looking forward to make this very accessible and customizable for you so keep tuned with **Shady's** updates!

**Comment by tougrel** [comment]
I'm closing this because it's outdated.
